infoschema/infoschema_test.go

```diff
@@ -122,13 +122,13 @@ func (*testSuite) TestT(c *C) {
 	is := handle.Get()
 
 	schemaNames := is.AllSchemaNames()
-	c.Assert(schemaNames, HasLen, 3)
-	c.Assert(testutil.CompareUnorderedStringSlice(schemaNames, []string{infoschema.Name, "PERFORMANCE_SCHEMA", "Test"}), IsTrue)
+	c.Assert(schemaNames, HasLen, 4)
+	c.Assert(testutil.CompareUnorderedStringSlice(schemaNames, []string{infoschema.Name, "METRIC", "PERFORMANCE_SCHEMA", "Test"}), IsTrue)
```

Comment:
It's better to define a constant for `METRICS_SCHEMA`.

Reply:
Great, done.